test/distributed/test_c10d_nccl.py

```diff
@@ -1357,6 +1357,22 @@ def test_comm_split_subgroup(self):
         self.assertEqual(backend.comm_split_count(), 1)
         self.assertEqual(tensor, original_tensor)
 
+    @requires_nccl_version((2, 18), "Need NCCL 2.18+ for ncclCommSplit")
```

Comment:
will update the decorator here in the final diff ncclCommSplit is not needed here

Reply:
maybe break this into two tests, one for eager, one for eager async?  the eager is implicitly tested in `test_comm_split_subgroup` but separating it out from the actual split testing would help if something actually broke the eager init directly

maybe better since we have a full test matrix of (eager_or_lazy, blocking_or_nonblocking) that we test all the combos in a single test case?